integration-tests/cucumber-tests-platform-publiclighting/src/test/java/org/opensmartgridplatform/cucumber/platform/publiclighting/domain/platform/AbstractPlatformDeviceCreator.java

```diff
@@ -41,14 +39,16 @@ public abstract class AbstractPlatformDeviceCreator<T extends Device>
 
   @Autowired private DomainInfoRepository domainInfoRepository;
 
-  protected InetAddress networkAddress(final Map<String, String> settings) {
-    try {
-      return InetAddress.getByName(
-          ReadSettingsHelper.getNullOrNonEmptyString(
-              settings, PlatformKeys.KEY_NETWORKADDRESS, PlatformDefaults.DEFAULT_NETWORK_ADDRESS));
-    } catch (final UnknownHostException e) {
-      throw new InvalidArgumentException("Invalid hostname");
-    }
+  @Value("${device.networkaddress}")
```

Comment:
Where in the properties is this value located? Because the value is not within this package. I can only find the value here: https://github.com/OSGP/open-smart-grid-platform/blob/ecb7dda7272cdc5f8bf9129dbf7192929e984063/integration-tests/cucumber-tests-platform/src/test/resources/cucumber-tests-platform.properties#L44

Reply:
It should be set in a local properties file, see the added README in the accompanying PR in the docker repo.

Reply:
Could we maybe make it visible for open source users or is that too much to ask?